demo.md

```diff
@@ -24,9 +24,10 @@ Before proceeding to the section of your choice, please consider the following t
 * Testing is very important for us. Even if you don't write a single test (for instance, because of time constraints), your app should be easy to test (we love [Dependency injection](https://en.wikipedia.org/wiki/Dependency_injection)).
 * Error scenarios should be taken into consideration and it should be easy to add them, even if you don't explicitly handle them (e.g. using an `UIAlertController`).
 * Although UI and UX are important, we are more concerned in this demo with your thought process and with how you architect your application. Your demo should take into consideration features that might be added in the future.
-* You can use any 3rd party libraries you wish (`Alamofire`, `ReactiveCocoa`, `PromiseKit`, `Realm`, etc). In case you do, be prepared to justify why you chose them. You can use `CocoaPods`, `Carthage` or `Git Submodules` for this (please don't drag and drop stuff around).
+* You can use any 3rd party libraries you wish (`Alamofire`, `ReactiveCocoa`, `PromiseKit`, `Realm`, etc) but be prepared to justify why you did so. Feel free to use package managers to handle them.
 * **Be consistent in your code**. We advise using something like [raywenderlich's swift style guide](https://github.com/raywenderlich/swift-style-guide) while doing the demo. It's absolutly fine to use any other style, as long as you are consistent.
-* Clean the file project structure (if you are creating one from scratch) and remove any unused methods (e.g., from `AppDelegate`). This shows attention to detail.
+* Clean the file project structure and remove any unused methods (e.g., from `AppDelegate`). This shows attention to detail.
+* If you have to use VIPER please give it your own spin. Even though it's a good architecture often times it carries a lot of boilerplate which makes all tests that use it look the same.
```

Comment:
"If you have to use VIPER please give it your own spin. Even though it's a good architecture often times it carries a lot of boilerplate which makes all tests that use it look the same."

I don't think we should have this sentence.